scenariolib/event_click.go

```diff
@@ -77,15 +77,15 @@ func newClickEvent(e *JSONEvent) (*ClickEvent, error) {
 
 // Execute Execute the click event, sending a click event to the usage analytics
 func (ce *ClickEvent) Execute(v *Visit) error {
-	if v.LastResponse.TotalCount < 1 {
-		if ce.fakeClick {
-			v.LastResponse = &ce.fakeResponse
-		} else {
-			Warning.Printf("Last query %s returned no results cannot click", v.LastQuery.Q)
-			return nil
-		}
-
+	if ce.fakeClick {
+		searchUID := v.LastResponse.SearchUID
+		v.LastResponse = &ce.fakeResponse
+		v.LastResponse.SearchUID = searchUID
+	} else if v.LastResponse.TotalCount < 1 {
```

Comment:
What? why is it a else if to check if clicks returned data?
If your fakeResponse is correctly built, it should have a TotalCount?